README.md

```diff
@@ -60,3 +76,15 @@ Steps:
 1. Click the ``Open in Gitpod`` button.
 2. Relax: a development environment with an active Wagtail Guide site will be created for you.
 3. Login at `/admin/` with username `admin` and password `changeme`
+
+
+# Contributing
+
+If you're a python or Django developer, fork the repo and join us. You'll find answers to many common new contributor questions in our [contributing guidelines](https://github.com/wagtail/bakerydemo/blob/master/contributing.md).
+
+# Other Notes
+
+## Google Summer of Code
+
+This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/). Find more details about the project [here](https://summerofcode.withgoogle.com/programs/2022/projects/7nMw2hTq).
```

Comment:
This section needs attention.

- Link `Google Summer of Code` to `https://summerofcode.withgoogle.com/`
- Mention Hitansh as contributor.
- Mention the rest of the team (?)
- The `click-here-link` is terrible, and Wagtail.org is not about "THIS project" (the guide). We still should link Wagtail.
- Maybe include https://github.com/wagtail/wagtail/wiki/Google-Summer-of-Code-2022